langchain/package.json

```diff
@@ -2166,5 +2225,8 @@
       "require": "./schema/prompt_template.cjs"
     },
     "./package.json": "./package.json"
+  },
+  "resolutions": {
```

Comment:
What is this?

Reply:
For the circular dep, not sure I need it anymore though. Will try without it